lib/chainer/functions/connection/embed_id.rb

```diff
@@ -0,0 +1,50 @@
+module Chainer
+  module Functions
+    module Connection
+      class EmbedIDFunction < Chainer::Function
+        def initialize(ignore_label: nil)
+          @ignore_label = ignore_label
+        end
+
+        def self.embed_id(x, w, ignore_label: nil)
+          self.new(ignore_label: ignore_label).(x, w)
+        end
+
+        def forward_cpu(inputs)
```

Comment:
I am glad if you can support `Cumo` together.

Codes would become like:


```
def forwrad(inputs)
  xm = Chainer.get_array_module(*inputs)
```

and replace `Numo` with `xm`